common/v2/components/AccountList.tsx

```diff
@@ -159,7 +169,9 @@ export default function AccountList(props: AccountListProps) {
     return (
       <AccountListFooterWrapper>
         <RouterLink to={ROUTE_PATHS.ADD_ACCOUNT.path}>
-          <Typography>{`+ ${translateRaw('ACCOUNT_LIST_TABLE_ADD_ACCOUNT')}`}</Typography>
+          <AddAccountButton basic={true}>{`+ ${translateRaw(
```

Comment:
what is the `basic` attr for?

Reply:
I copied it from the Address Book component, as they were implemented differently but are essentially the same (add account to account table vs add address to address book table). I figured a button was more appropriate than type, given the context of the user clicking `+ Add Account`. I think the `basic` may produce an unstyled button (no blue background, etc.) as I didn't have to override any styling.